WEB_SERVICE/WEB/_HIGH/PATH_TRAVERSAL/recommendation.md

```diff
@@ -0,0 +1,153 @@
+To mitigate the risk of path traversal, consider the following recommendations:
+
```

Comment:
From portswigger and the one that I suggest : "After validating the supplied input, append the input to the base directory and use a platform filesystem API to canonicalize the path. Verify that the canonicalized path starts with the expected base directory."